tests/fixtures/module-unification-addon/package.json

```diff
@@ -40,13 +40,12 @@
     "ember-load-initializers": "^1.0.0",
     "ember-maybe-import-regenerator": "^0.1.6",
     "ember-resolver": "^4.0.0",
-    "ember-source": "~3.1.0-beta.1",
+    "ember-source": "http://builds.emberjs.com/canary.tgz",
```

Comment:
This URL is out of date. I don't think we should ever be using it, /cc @rwjblue yes?

Reply:
Also at https://github.com/ember-cli/ember-cli/blob/e9a7b587a1ebd063985816330fd1a193652cec54/blueprints/module-unification-app/files/package.json#L39 for MU apps.

Reply:
Confirm. We need to fetch the current URL each time (via `ember-source-channel-url` which returns a promise)

Reply:
@rwjblue Any guidance how to get the `ember-source-channel-url` at build time? Specially about how to handle it in testing, where we shouldn't rely on the network.